logos/tests/langchainTest.py

```
@@ -0,0 +1,107 @@
+import json
+import logging
+from pprint import pprint
+
+import langchain_core
+from langchain.agents import create_tool_calling_agent, AgentExecutor
+from langchain_core.language_models.chat_models import SimpleChatModel
+from langchain_core.messages import AIMessage, HumanMessage, BaseMessage, SystemMessage
+from langchain_core.prompt_values import ChatPromptValue
```

Ruff (0.11.9)

`langchain_core.prompt_values.ChatPromptValue` imported but unused

Remove unused import: `langchain_core.prompt_values.ChatPromptValue`

(F401)